### DISCUSSION TITLE
Recomiendeme herramientas que deba utilizar con GitHub Education

### DISCUSSION THREAD

**Discussion by Yerridev** [community/community#126183] [GitHub Education]
### Hi everyone,

Veo una gran cantidad de herramientas por lo que me parece interesante recibir recomendaciones.

**Comment by davevad93** [comment]
Hi @Yerridev ,

It depends on what interests you the most, the [GitHub Student Developer Pack](https://education.github.com/pack/offers) includes many different things, you should carefully explore everything it has to offer. For example if you're into Front-end development you can connect your GitHub account to [FrontEndMasters](https://frontendmasters.com/) with 6-months access to all courses and workshops, or if you're interested in CI/CD you have access to [Travis CI](https://www.travis-ci.com/) which offers private builds for free while you're a student. It also offers access to many Microsoft tools, such as [Azure Cloud](https://azure.microsoft.com/es-es) or [Microsoft Dev Essentials](https://visualstudio.microsoft.com/es/dev-essentials/). Obviously the first thing I recommend is to take advantage of the free subscriptions to Copilot and JetBrains but as I said, there's a lot to choose from and that's up to you. Hope this helps!
